plugins/modules/sp_client_policy.py

```diff
@@ -0,0 +1,178 @@
+        description: Type of the schedule (clientaction/admin).
+        required: true
+        type: str
+        choices: ["clientaction", "admin"]
+    start_time:
+        description: Schedule start time (HH:MM:SS format).
+        required: true
+        type: str
+    period:
+        description: Period between backups.
+        required: true
+        type: int
+    period_unit:
+        description: Unit of the period (hours/days).
+        required: true
+        type: str
+        choices: ["hours", "days"]
+    client_nodes:
+        description: List of client nodes to associate with the schedule.
+        required: true
+        type: list
+        elements: str
+extends_documentation_fragment: ibm.storage_protect.auth
+...
+'''
+
+
+EXAMPLES = '''
+- name: Configure backup schedule for client nodes
+  ibm.storage_protect.sp_client_policy:
+    schedule_name: "{{ DailyBackup }}"
+    schedule_type: "{{ clientaction }}"
+    start_time: "22:00:00"
+    period: 1
+    period_unit: "days"
+    client_nodes:
+      - "node1"
+      - "node2"
+...
+'''
+
+from ..module_utils.dsmadmc_adapter import DsmadmcAdapter
+
+
+def main():
+    argument_spec = dict(
+        schedule_name=dict(type="str", required=True),
+        schedule_type=dict(type="str", required=True, choices=["clientaction", "admin"]),
+        start_time=dict(type="str", required=True),
+        period=dict(type="int", required=True),
+        period_unit=dict(type="str", required=True, choices=["hours", "days"]),
+        client_nodes=dict(type="list", elements="str", required=True)
+    )
+
+    required_by = {
+        'backup_repl_rule_default': 'replication_state',
+        'archive_repl_rule_default': 'replication_state',
+        'space_repl_rule_default': 'replication_state',
+        # 'schedules': 'policy_domain',
+    }
+
+    module = DsmadmcAdapter(argument_spec=argument_spec, supports_check_mode=True, required_by=required_by)
+    
+
+    schedule_name = module.params["schedule_name"]
+    schedule_type = module.params["schedule_type"]
+    start_time = module.params["start_time"]
+    period = module.params["period"]
+    period_unit = module.params["period_unit"]
+    client_nodes = module.params["client_nodes"]
+
+    schedule_name = module.params.get('schedule_name')
+    state = module.params.get('state')
+    exists, existing = module.find_one('sp_client_policy', schedule_name)
+
+    if state == 'absent' or state == 'deregistered' or state == 'removed':
+        module.perform_action('remove', 'sp_client_policy', schedule_name, exists=exists)
+    else:
+        options_params = {
+            'schedule_type': 'Client',
+            'start_time': 'current_time',
+            'period': '1',
+            'period_unit': 'DAYS',
+            'client_nodes': 'client_nodes',
+        }
+
+        not_on_update = ['node_type', 'backup_repl_rule_default', 'archive_repl_rule_default', 'space_repl_rule_default', 'admin_user_id', 'option_set']
```

Comment:
I think the only one which can't change is type
```suggestion
        not_on_update = ['type']
```